CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+Although this project has dedicated engineers from Mesosphere working on modifications,
+community contributions are essential for keeping dcos-metrics great. In order to keep
+it as easy as possible for people to contribute, we have a few guidelines that we'd
+like contributors to follow.
+
+If you have any questions on the below guidelines, please feel free to contact us by using
+any of the community resources listed in this project's [README.md][dcos-metrics-readme] file.
+
+## Getting Started
+  * Make sure you have a [GitHub][github-join] account.
+  * Submit a ticket for your issue in the [DC/OS JIRA][dcos-jira], using the component
+  `dcos-metrics`.
+    * Clearly describe the issue and any steps you need to reproduce it.
+    * Be sure to include the version or SHA of the software you're using, and any
+    other information about your environment that might be relevant.
+  * Fork the repository on GitHub.
+
+## Making Changes
+  * Create a topic branch from where you want to base your work. Topic branches should typically
+  be based on the `master` branch.
+    * To quickly create a topic branch based on master, run `git checkout -b my-patch master`.
+    * Except for trivial changes, please avoid working directly on the `master` branch.
+  * Make commits of logical units. Ideally, commits could be tested and applied individually.
+
+### Trivial changes
+For trivial changes and documentation updates, it's not always necessary to create a new ticket
+or topic branch. Please, use your best judgement here.
+
+### Style guidelines
+Before submitting your pull request, please consider the following:
+  * Check for unnecessary whitespace by running `git diff --check`.
+  * Check for syntax and style issues by running `go fmt`.
+  * Include comments and examples where appropriate.
+  * No additional copyright statements or licenses.
```

Comment:
We should add in our preferred golang linters and ensure appropriate formatting.